module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,9 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> wordsList) {
+    wordsList.sort(std::less<std::string>());
```

Comment:
It's not a correct way of sorting. sort by default use std::less. You need to sort it by size and then "normally". 
Please fix it ;) Our tests do not detect such thing.

Reply:
updated